### ISSUE TITLE
Wwise RIFF header mistaken for Ubisoft Jade RIFF header

### ISSUE THREAD

**Issue by NicknineTheEagle** [kode54/vgmstream#227]
Ubisoft Jade RIFF header parser (https://github.com/kode54/vgmstream/blob/master/src/meta/ubi_jade.c) is placed higher in the list than Wwise RIFF parser (https://github.com/kode54/vgmstream/blob/master/src/meta/wwise.c) in vgmstream.c and the header appears to be similar enough that it accepts Wwise RIFF header and proceeds to play it with incorrect decoder.

It should either be moved below Wwise or have more checks added to it.

**Comment by bnnm** [comment]
All my Wwise samples work correctly so you'll need to be more specific.

**Comment by NicknineTheEagle** [comment]
The game where I ran into this problem is Doom 4, here is a sample audio chunk: [D4_MX_combat_industrial_03_163bpm_v2_HEAVY_13_3A67241E.zip](https://github.com/kode54/vgmstream/files/2018620/D4_MX_combat_industrial_03_163bpm_v2_HEAVY_13_3A67241E.zip)

It appears to be an extension conflict, both parsers accept WAV and LWAV extensions but it seems that both headers are extremely similar which causes Jade parser to accept and try to play a Wwise chunk.

The reason it became a problem to me is because the chunks from Doom 4 don't have filenames (or extensions) stored inside soundbanks, the only way to know them is to check soundbanksinfo.xml and *then* I figured out that the source files have WEM extension which does work correctly with vgmstream.

Still, this may be a problem in other games so I think it's worth looking into.
